colossalai/inference/pipeline/README.md

```diff
@@ -33,17 +33,22 @@ Pipeline Inference is composed of three parts: `PPInferEngine`, `MicroBatchManag
 ```python
 from colossalai.pipeline import PPInferEngine
 # Suppose the pipeline size is 2, and use fp16 to do infenrence. Use Llama as an example.
-model = LlamaForCausalLM.from_pretrained('/path/to/model')
-inputs = tokenizer("Hello, my dog is cute", "What a good day", return_tensors="pt")
-engine = PPInferEngine(
-    pp_size=2,
-    dtype='fp16',
-    micro_batch_size=1,
-    new_length=10,
-    model=model,
-    model_policy=LlamaForCausalLMPipelinePolicy())
-
-output = engine.inference([inputs])
+from colossalai.inference import PPInferEngine
+from colossalai.inference.pipeline.policies import LlamaModelInferPolicy
+import colossalai
+from transformers import LlamaForCausalLM, LlamaTokenizer
+
+colossalai.launch_from_torch(config={})
+
+model = LlamaForCausalLM.from_pretrained("path_to_model")
+tokenizer = LlamaTokenizer.from_pretrained("path_to_model")
+# assume the model is infered with 2 pipeline stages
+inferengine = PPInferEngine(pp_size=2, model=model, model_policy=LlamaModelInferPolicy(), new_length=8)
+
+input = ["Introduce a landmark in China ","Introduce a landmark in China "]
+data = tokenizer(input, return_tensors='pt')
+output = inferengine.inference([data.to('cuda').data])
```

Comment:
Is it possible to use “data.to('cuda')” directly?

Reply:
done